inspirehep/dojson/hepnames/fields/bd1xx.py

```diff
@@ -450,11 +451,15 @@ def advisors(self, key, value):
 
     _degree_type = force_single_element(value.get('g'))
     degree_type = DEGREE_TYPES_MAP.get(_degree_type, 'Other')
+    recid = force_single_element(value.get('x'))
```

Comment:
A space before this to separate the two different things the two pairs of lines are doing.